e2e/cli-e2e/tests/config.spec.ts

```diff
@@ -14,18 +16,61 @@ const configFile = (ext: Extension) =>
 const execCli = (argObj: Partial<CliArgsObject>) =>
   executeProcess({
     command: 'npx',
-    args: objectToCliArgs({
-      _: './dist/packages/cli',
-      verbose: true,
-      ...argObj,
-    }),
+    args: [
+      './dist/packages/cli',
+      'print-config',
+      ...objectToCliArgs({
+        verbose: true,
+        ...argObj,
+      }),
+    ],
   });
 
 // TODO: use print-config command once implemented and check stdout
-describe('CLI config parsing', () => {
+describe('print-config', () => {
   it.each(extensions)('should load .%s config file', async ext => {
-    const { code, stderr } = await execCli({ config: configFile(ext) });
+    const { code, stderr, stdout } = await execCli({ config: configFile(ext) });
     expect(code).toBe(0);
     expect(stderr).toBe('');
+    const args = JSON.parse(stdout);
+    expect(args).toEqual({
+      interactive: true,
+      verbose: true,
+      config: expect.stringContaining(`code-pushup.config.${ext}`),
+      upload: {
+        organization: 'code-pushup',
+        project: `cli-${ext}`,
+        apiKey: 'e2e-api-key',
+        server: 'https://e2e.com/api',
+      },
+      persist: { outputDir: `tmp/${ext}` },
+      plugins: expect.any(Array),
+      categories: expect.any(Array),
+    });
+  });
+
+  it('should load .%s config file and merge cli arguments', async () => {
```

Comment:
The `%s` only gets replaced when used with `.each`.

```suggestion
  it('should load .ts config file and merge cli arguments', async () => {
```